src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -10,15 +10,19 @@ public enum Category
 
 	public static class Descriptors
 	{
+		private const string DocumentationLinkPrefix = @"https://github.com/Acumatica/Acuminator/blob/dev/docs/diagnostics";
```

Comment:
Url should be `https://github.com/Acumatica/Acuminator/docs/diagnostics`

Reply:
fixed